### PR TITLE
Fix assigment of NA_REAL to LogicalVector

### PR DESCRIPTION
**Pull request by zeehio**
Catched by https://cran.r-project.org/web/checks/check_results_dplyr.html (undefined behaviour sanitizer)

### CODE REVIEW

**Comment by krlmlr** [comment]
Awesome, thanks. Could you add a NEWS bullet, please?

**Comment by zeehio** [comment]
Thanks for the review. Done :+1:

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# dplyr 0.7.0.9000
+
+## Bug fixes
```
Comment (hadley) [inline NEWS.md]: Can you remove this please? It's easier to figure out what the second level headings should be just before release

Comment (zeehio) [inline NEWS.md]: Done. :+1:

**Comment by hadley** [comment]
Thanks!

**Pull request merged**